src/render/graph.rs

```diff
@@ -417,14 +417,6 @@ impl Graph {
             // acquire a mutable borrow of the current processing node
             let mut node = nodes[*index].borrow_mut();
 
-            // make sure all input buffers have the correct number of channels, this might not be
```

Comment:
This is the main change. I think I just misunderstood the spec 'channelCount' behaviour